install.sh

```diff
@@ -32,12 +35,12 @@ if [ ! -f /etc/profile.d/docker-uid.sh ]; then
 fi
 
 # Build docker images
-docker-compose build .
+docker-compose build
```

Comment:
Removed this `.` since docker-compose looks for it's yml file in the default location.